openapiart/openapiartgo.py

```diff
@@ -1045,6 +1050,12 @@ def _get_version_api_interface_method_impl(self, struct_name):
                 api.versionMeta.checkVersion = v
             }}
 
+            func (api *{0}) SetComponentInformation(clientName string, clientVer string, serverName string) {{
```

Comment:
is it not possible to get `clientVer` automatically from Snappi SDK. or it has any specific use case?